packages/jest-matcher-utils/src/index.js

```diff
@@ -113,37 +113,31 @@ const stringify = (object: any, maxDepth?: number = 10): string => {
 const highlightTrailingWhitespace = (text: string, bgColor: Function): string =>
   text.replace(/\s+$/gm, bgColor('$&'));
 
-const printReceived = (object: any) => highlightTrailingWhitespace(
-  RECEIVED_COLOR(stringify(object)),
-  RECEIVED_BG,
-);
-const printExpected = (value: any) => highlightTrailingWhitespace(
-  EXPECTED_COLOR(stringify(value)),
-  EXPECTED_BG,
-);
+const printReceived = (object: any) =>
+  highlightTrailingWhitespace(RECEIVED_COLOR(stringify(object)), RECEIVED_BG);
+const printExpected = (value: any) =>
+  highlightTrailingWhitespace(EXPECTED_COLOR(stringify(value)), EXPECTED_BG);
 
 const printWithType = (
   name: string,
   received: any,
   print: (value: any) => string,
 ) => {
   const type = getType(received);
-  return (
-    name + ':' +
-    (type !== 'null' && type !== 'undefined'
-      ? '\n  ' + type + ': '
-      : ' ') +
-    print(received)
-  );
+  return name +
+    ':' +
+    (type !== 'null' && type !== 'undefined' ? '\n  ' + type + ': ' : ' ') +
```

Comment:
Honestly, the initial version is also pretty crazy. I would consider refactoring it.